CWE/inventions/culture_inventions.txt

```diff
@@ -629,117 +666,123 @@ social_anthropology = {
+		value = 0.50
+		}
+	}
 }
 man_in_orbit = {
-        limit = { cold_war = 1 }
-        chance = {
-                base = 2
-                modifier = {
-                    factor = 12
-					OR = { 
-						tag = USA 
-						tag = RUS 
-					} 
-                }
-                modifier = {
-                        factor = 2
-                        is_greater_power = yes
-                }
-        }
-        effect = {
-            permanent_prestige = 2
-			education_efficiency = 0.1
-        }
+	limit = { cold_war = 1 }
+	news = no
+	chance = {
+		base = 2
+		modifier = {
+			factor = 12
+			OR = { 
+				tag = USA 
+				tag = RUS 
+			} 
+		}
+		modifier = {
+			factor = 2
+			is_greater_power = yes
+		}
+	}
+	effect = {
+		permanent_prestige = 2
+		education_efficiency = 0.1
+	}
 }
 spacewalk = {
-        limit = { cold_war = 1 }
-        chance = {
-                base = 2
-                modifier = {
-                    factor = 12
-					OR = { 
-						tag = USA 
-						tag = RUS 
-					} 
-                }
-                modifier = {
-                        factor = 2
-                        is_greater_power = yes
-                }
-        }
-        effect = {
-            permanent_prestige = 2
-			education_efficiency = 0.1
-        }
+	limit = { cold_war = 1 }
+	news = no
+	chance = {
+		base = 2
+		modifier = {
+			factor = 12
+			OR = { 
+				tag = USA 
+				tag = RUS 
+			} 
+		}
+		modifier = {
+			factor = 2
+			is_greater_power = yes
+		}
+	}
+	effect = {
+		permanent_prestige = 2
+		education_efficiency = 0.1
+	}
 }
 space_docking = {
-        limit = { detente = 1 }
-        chance = {
-                base = 2
-                modifier = {
-                    factor = 12
-					OR = { 
-						tag = USA 
-						tag = RUS 
-					} 
-                }
-                modifier = {
-                        factor = 2
-                        is_greater_power = yes
-                }
-        }
-        effect = {
-            permanent_prestige = 2
-			education_efficiency = 0.1
-        }
+	limit = { detente = 1 }
+	news = no
+	chance = {
+		base = 2
+		modifier = {
+			factor = 12
+			OR = { 
+				tag = USA 
+				tag = RUS 
+			} 
+		}
+		modifier = {
+			factor = 2
+			is_greater_power = yes
+		}
+	}
+	effect = {
+		permanent_prestige = 2
+		education_efficiency = 0.1
+	}
 }
 space_station_invention = {
-        limit = { detente = 1 }
-        chance = {
-                base = 2
-                modifier = {
-                    factor = 12
-					OR = { 
-						tag = USA 
-						tag = RUS 
-					} 
-                }
-                modifier = {
-                        factor = 2
-                        is_greater_power = yes
-                }
-        }
-        effect = {
-            permanent_prestige = 2
-			education_efficiency = 0.1
-        }
+	limit = { detente = 1 }
+	news = no
+	chance = {
+		base = 2
+		modifier = {
+			factor = 12
+			OR = { 
+				tag = USA 
+				tag = RUS 
+			} 
```

Comment:
Should check for space agency country modifier instead of tags to avoid rail roading

Reply:
Alright, these were all from the old inventions - I haven't gone through and edited the modifiers for them except for the ones that were not in line with later ones. I can change all those instances as well as any others if you find some that also look wonky. Its hard to go through them all at once.

Reply:
It marked all these as something I added because @Outta-my-way-burrocrat and myself altered other parts of the file and the count changed.

Reply:
Yes please, thanks!